CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,115 @@
+# CFPB Open Source Code of Conduct
+
+## Introduction
+
+The [Consumer Financial Protection Bureau](https://www.consumerfinance.gov) (CFPB) is committed to 
+building a safe, welcoming, harassment-free culture for everyone. We do not merely want an 
+environment that is free from hostility, we want one that is actively welcoming and inclusive. We 
+want our team, our workplace culture, and our open source community to reflect and celebrate the 
+diversity of the people we serve.
+
+This Code of Conduct summarizes federal anti-harassment law and CFPB policy. 
+
+## Scope
+
+We expect everyone on the CFPB team, and those contributing to our open source community, to exhibit 
+these behaviors and abide by applicable federal laws and CFPB policies. In addition, we expect 
+everyone within CFPB spaces to exhibit these behaviors and refrain from behavior prohibited by 
+anti-harassment laws and federal policies on harassment. These spaces include:
+
+- CFPB’s physical offices,
+- CFPB events and meetings,
+- All of CFPB’s online forums and virtual collaboration tools, including code repositories
```

Comment:
Bulleted list should either omit trailing commas, end in periods, or have a trailing period for the last item.